### ISSUE TITLE
Unable to start up parallels provider since 1.7.4

### ISSUE THREAD

**Issue by pieterjvrensburg** [Parallels/vagrant-parallels#297]
I upgraded to version 1.7.4 and now I am unable to start up.

````
➜  centos-6.8 vagrant init parallels/centos-6.8; vagrant up --provider parallels
A `Vagrantfile` has been placed in this directory. You are now
ready to `vagrant up` your first virtual environment! Please read
the comments in the Vagrantfile as well as documentation on
`vagrantup.com` for more information on using Vagrant.
/opt/vagrant/embedded/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require': cannot load such file -- nokogiri (LoadError)
	from /opt/vagrant/embedded/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	from /Users/pieter.jvrensburg/.vagrant.d/gems/2.2.5/gems/vagrant-parallels-1.7.4/lib/vagrant-parallels/driver/base.rb:2:in `<top (required)>'
	from /Users/pieter.jvrensburg/.vagrant.d/gems/2.2.5/gems/vagrant-parallels-1.7.4/lib/vagrant-parallels/driver/meta.rb:4:in `require_relative'
	from /Users/pieter.jvrensburg/.vagrant.d/gems/2.2.5/gems/vagrant-parallels-1.7.4/lib/vagrant-parallels/driver/meta.rb:4:in `<top (required)>'
	from /Users/pieter.jvrensburg/.vagrant.d/gems/2.2.5/gems/vagrant-parallels-1.7.4/lib/vagrant-parallels/provider.rb:16:in `usable?'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.5/lib/vagrant/vagrantfile.rb:138:in `machine_config'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.5/lib/vagrant/vagrantfile.rb:45:in `machine'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.5/lib/vagrant/environment.rb:669:in `machine'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.5/lib/vagrant/plugin/v2/command.rb:177:in `block in with_target_vms'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.5/lib/vagrant/plugin/v2/command.rb:201:in `call'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.5/lib/vagrant/plugin/v2/command.rb:201:in `block in with_target_vms'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.5/lib/vagrant/plugin/v2/command.rb:183:in `each'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.5/lib/vagrant/plugin/v2/command.rb:183:in `with_target_vms'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.5/plugins/commands/up/command.rb:131:in `install_providers'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.5/plugins/commands/up/command.rb:85:in `execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.5/lib/vagrant/cli.rb:42:in `execute'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.5/lib/vagrant/environment.rb:308:in `cli'
	from /opt/vagrant/embedded/gems/gems/vagrant-1.9.5/bin/vagrant:138:in `<main>'
````

**Comment by legal90** [comment]
@pieterjvrensburg Hi! What version Vagrant do you use? 
Try to reinstall the plugin:
```
vagrant plugin uninstall vagrant-parallels
vagrant plugin install vagrant-parallels
```
Does it help?

**Comment by legal90** [comment]
These issues look relevant:
https://github.com/mitchellh/vagrant/issues/8593
https://github.com/mitchellh/vagrant/issues/8594
https://github.com/mitchellh/vagrant/issues/8595

**Comment by pieterjvrensburg** [comment]
I am running the latest, also updated before getting to this point.

````
➜  centos-6.9 vagrant --version
Vagrant 1.9.5
````

I have tried uninstalling and re-installing the plugin. Yesterday I uninstalled everything and started fresh, but will try the above suggestions.

**Comment by pieterjvrensburg** [comment]
Hi @legal90,

Thank you for the links, #8594 worked for me. This is what I did:

````
➜  centos-6.9 NOKOGIRI_USE_SYSTEM_LIBRARIES=true vagrant plugin install nokogiri 
Installing the 'nokogiri' plugin. This can take a few minutes...
Building native extensions.  This could take a while...
Installed the plugin 'nokogiri (1.7.2)'!

➜  centos-6.8 vagrant up --provider parallels 
WARNING: Nokogiri was built against LibXML version 2.9.4, but has dynamically loaded 2.9.2
Bringing machine 'default' up with 'parallels' provider...
````

Is the warning something to worry about?

**Comment by legal90** [comment]
That issue was fixed by GH-298, the fix was released in vagrant-parallels **v1.7.5**.


### How to use `vagrant-parallels` plugin with Vagrant 1.9.5
1. Remove it (if it was previously installed):
```bash
$ vagrant plugin uninstall vagrant-parallels
```
2. Install it with `NOKOGIRI_USE_SYSTEM_LIBRARIES` env variable defined:
```bash
$ NOKOGIRI_USE_SYSTEM_LIBRARIES=true vagrant plugin install vagrant-parallels
```


Done. After that `vagrant-parallels` plugin should work fine. The only thing you might be confused about is the LibXML warning like this one:
```
$ vagrant up
WARNING: Nokogiri was built against LibXML version 2.9.4, but has dynamically loaded 2.9.2
Bringing machine 'default' up with 'parallels' provider...
...
```
Just disregard this warning for now. It's caused by the difference between LibXML version bundled with Vagrant and the one installed on your macOS. However, it should not affect the plugin's operation.

**Comment by legal90** [comment]
UPDATE: The problem was solved in Vagrant 1.9.7:

```
$ vagrant plugin install vagrant-parallels
Installing the 'vagrant-parallels' plugin. This can take a few minutes...
Fetching: mini_portile2-2.2.0.gem (100%)
Fetching: nokogiri-1.8.0.gem (100%)
Building native extensions.  This could take a while...
Fetching: vagrant-parallels-1.7.5.gem (100%)
```
